src/pvrclient-argustv.cpp

```diff
@@ -736,9 +740,12 @@ PVR_ERROR cPVRClientArgusTV::GetRecordings(ADDON_HANDLE handle)
 PVR_ERROR cPVRClientArgusTV::DeleteRecording(const PVR_RECORDING &recinfo)
 {
   PVR_ERROR rc = PVR_ERROR_FAILED;
+  std::string UNCname, strRecId = recinfo.strRecordingId;
 
-  XBMC->Log(LOG_DEBUG, "->DeleteRecording(%s)", recinfo.strRecordingId);
-  std::string UNCname = ToUNC(recinfo.strStreamURL);
+  if (!FindRecEntryUNC(strRecId, UNCname))
```

Comment:
You should be able to optimize this a little: `if (!FindRecEntryUNC(recinfo.strRecordingId, UNCname))`. You actually don't need the `strRecId` variable.

Applies to some more places in your PR.